patches/minecraft/net/minecraft/client/multiplayer/ClientPacketListener.java.patch

```diff
@@ -95,3 +95,7 @@
           }
        } finally {
           if (friendlybytebuf != null) {
++            if (false) // Forge: let packet handle releasing buffer
```

Comment:
I somewhat understand why we can't release the buffer and have to revert the patch down below, but this patch should not be needed, as this is just releasing the copied buffer

Reply:
What do you mean with: "this patch is not needed"?

Reply:
This PR adds the if (false) patch again, but this patch should have been omitted.
I just mean that we only need a partial revert, the ClientboundCustomPayloadPacket patch needs to be reverted, but the removal of the if (false) line in #8042 was correct, so this if (false) should NOT be added again

Reply:
I think I will have to leave this in. The point is that the buffer can not be released in a LAN world.

However I will need to check this later

Reply:
This buffer is fine to be released, as this is obtained from `ClientboundCustomPayloadPacket#getData()` which returns a `FriendlyByteBuf` containing a copy of the data from the packet's own buffer. So the patch should be removed (this addition be omitted from the PR), as this prevents the copied buffer from being released properly.

Reply:
Okiedokie